### DISCUSSION TITLE
Delet my website

### DISCUSSION THREAD

**Discussion by rachidsahli** [quarto-dev/quarto-cli#10345] [Q&A] [answered]
### Description

Hi, i've deleted all the code I used to create my site. I don't have it on github or on my computer. It is however published on quartopub. Is there any way of recovering the code ?

**Comment by mcanouil** [comment, marked as answer]
No.
Unless you used the `code-tools` option to have the code for each pages.
Even with this option you won't have project and directory metadata files.

QuartoPub only has the HTML files not the source.

**Comment by rachidsahli** [comment]
Ok, thanks
